css/main.css

```diff
@@ -34,6 +34,70 @@ label.dynamic-info {
 	font-style: italic;
 }
 
+input[type="number"] {
+	max-width: 7em;
+
+	/* Remove spinner controls from Firefox */
+	appearance: textfield;
+}
+
+/* Re-applies the controls on :hover and :focus
+to get the same behavior as on chrom* */
+input[type="number"]:hover,
+input[type="number"]:focus {
+	appearance: number-input;
+}
+
+select[data-provides="employee"] {
+	width: 6.5em;
+}
+
+/* Enables the first input's help text to pop
+over the lasts of this group.
+Also keeps related inputs together. */
+#input fieldset {
+	white-space: nowrap;
+	display: inline-block;
+}
+
+/* Reset fieldset style */
+fieldset {
+	border: 0;
+	padding: 0;
+	padding-top: .01em;
+	margin: 0;
+	min-width: 0;
+}
+
+input#salaire:hover ~ *,
+input#salaire:focus ~ * {
+	visibility: hidden;
+}
+input#salaire:hover + label,
+input#salaire:focus + label {
+	visibility: visible;
+}
+input#salaire:hover ~ .input-help,
+input#salaire:focus ~ .input-help {
```

Comment:
Then you have a badly-named ID  :wink:  Considering an ID has to be unique across page, if it's ambiguous in naming it's failing its purpose.